### PR TITLE
add img2pdf

### PR DESCRIPTION
**Pull request by dougalsutherland**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dougalsutherland** [comment]
```
+ source run_conda_forge_build_setup
/opt/conda/bin/run_conda_forge_build_setup: line 4: CI: unbound variable
##[error]Bash exited with code '1'.
```
???

**Comment by chrisburr** [comment]
@conda-forge-admin, please lint

File: recipes/img2pdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d77c17ee65a736abe92ef8cba9cca009c064ea4ed74492c01aea596e41856cf
+
+build:
+  number: 0
+  noarch: python
```
Comment (chrisburr) [inline recipes/img2pdf/meta.yaml]: Unfortunately the `enum34` dependency means this can't be noarch
```suggestion
```

**Comment by dougalsutherland** [comment]
Yeah, I know. :)  (FYI, you can make an empty commit with `git commit --allow-empty -m "empty commit to force CI"`.)

I guess the earlier CI failure was something weird and intermittent, rerunning just to make sure.

**Comment by chrisburr** [comment]
Sorry about that. I had just fixed the CI issue and then went through the failed PRs to retrigger them (it was easier to do it with minor changes in web interface instead of setting up lots of remotes). I didn't check closely enough when deciding to make it noarch and then immediately forgot that I made the change. 😆

**Pull request merged**